### PR TITLE
Add podspec file

### PR DESCRIPTION
**Pull request by keidev**

### CODE REVIEW

File: native/ios/WatermelonDB/SupportingFiles/Bridging.h [hunk 1]
```diff
@@ -1,2 +1,2 @@
-#import "../FMDB/src/fmdb/FMDB.h"
+#import "FMDB.h"
```
Comment (radex) [inline native/ios/WatermelonDB/SupportingFiles/Bridging.h]: that will break watermelon when not using cocoa pods

Comment (keidev) [reply native/ios/WatermelonDB/SupportingFiles/Bridging.h]: @radex This compiled with Xcode 11.2.
I don’t know how to keep the folder structure in Cocoa pods.
See CocoaPods/CocoaPods#8873

Comment (radex) [reply native/ios/WatermelonDB/SupportingFiles/Bridging.h:1]: @sidferreira sure, give it a try

**Comment by radex** [comment]
closing in favor of https://github.com/Nozbe/WatermelonDB/pull/731

**Pull request closed without merging**